src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -557,7 +563,10 @@ private void setupBookingAccount(final OffenderBooking booking, final AgencyLoca
     private void setYouthStatus(final OffenderBooking booking) {
         final var profileType = profileTypeRepository.findByTypeAndCategoryAndActive("YOUTH", "PI", true).orElseThrow(EntityNotFoundException.withId("YOUTH"));
         final var profileCode = profileCodeRepository.findById(new ProfileCode.PK(profileType, "Y")).orElseThrow(EntityNotFoundException.withMessage("Profile Code for YOUTH and Y not found"));
-        booking.add(profileType, profileCode);
+        List<OffenderProfileDetail> profileDetails = offenderProfileDetailRepository.findByOffenderBookingBookingIdAndTypeAndSequence(booking.getBookingId(), profileType, profileCode.getListSequence());
+        if (profileDetails.isEmpty()) {
```

Comment:
think will need some tidying up if this is the way we want to go.. but happy to approve for now so you can test in dev. Might be best to get a revert PR ready in case people need to release..